.yamato/config/__shared.metafile

```diff
@@ -16,16 +16,6 @@ editors: # applied for all yml files
     allow_triggers: True
     variables:
       CACHE_ARGS: '{{cache.flags}}'
-  - track: 2021.2
-    name: 2021.2
-    rerun_strategy: on_new_revision
-    ono_branch_var: ONO_BRANCH_2021_2
-    editor_pinning:
-      enabled: True
-      nr_commits_to_retrieve: 100
```

Comment:
We should probably set this `nr_commits_to_retrieve: 100` to the trunk editor now, for evergreen bisections